logos/logos-orchestrator/src/logos/main.py

```
@@ -806,11 +807,59 @@ def _capture_logosnode_provider_snapshot(
                         _resolve_provider_name(provider_id),
                         exc_info=True,
                     )
+                else:
+                    _reconcile_model_profiles_across_nodes(db)
 
     sample["snapshot_id"] = snapshot_id
     asyncio.create_task(_logosnode_registry.record_runtime_sample(provider_id, sample))
```

Ruff (0.16.1)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)